server/Server.js

```diff
@@ -69,48 +69,80 @@ class Server {
         console.log("");
     }
 
-    captureEvents() {
-        // Capturing every post event sent to this server
-        const snowplowMicroServer = this.snowplowMicroServer;
-        this.instance.post("*", async function (req, res) {
-            var badEvents = [];
-            var goodEvents = [];
+    async handleEvent(request, response) {
+        const body = request.body;
+        const bundle = body.data.reverse();
 
-            try {
-                await snowplowMicroServer.validateEvent(req.body);
-                const [badEventResponse, goodEventResponse] = await Promise.all([
-                    snowplowMicroServer.retrieveBadEvents(),
-                    snowplowMicroServer.retrieveGoodEvents(),
-                ]);
+        var badEvents = [];
```

Comment:
Since we are touching it here, do you think it's worth to make it `let`? I have a `var` phobia as it has the weirdest behavior one can imagine.

Reply:
Will make sure to add a linter to fix this 👍